src/components/common/Tipping/render/Summary/Summary.tsx

```diff
@@ -49,6 +50,19 @@ export const Summary: React.FC<SummaryProps> = props => {
           />
         </Typography>
       </div>
+      {receiver.walletDetail.referenceType !== 'wallet_address' && (
+        <div className={styles.warningNoNear}>
+          <div className={styles.wrapperIcon}>
+            <InfoIconYellow />
+          </div>
+          <Typography className={styles.textWarning}>
+            The tip will be stored in Myriad Escrow because the user hasn’t connected the
+            {` ${currency.symbol} `}
```

Comment:
Please change `currency.symbol` 
The wallet is something like this [MYRIAD APP](https://app.testnet.myriad.social/wallet?type=manage)
Network MYRIAD, KUSAMA, and POLKADOT using  polkadot{.js}
Network NEAR using NEAR wallet